data/emulator.js

```diff
@@ -452,7 +452,10 @@ class EmulatorJS {
         this.elements.parent.appendChild(this.textElem);
     }
     localization(text, log) {
-        if (!isNaN(text)) return text;
+        if (!text) return text;
+        text = text.toString();
+        if (text === " ") text = "space";
```

Comment:
what does this do?

Reply:
if it equals a space it changes the text to space, it is for the control mapping.

Reply:
but what does it do overall, how does it affect the outcome?

Reply:
Because if you can map the space bar then it will show a space not the actual space text.

Reply:
Oh, can we change this in the keychange event then? We shouldnt do this here

Reply:
Ok I'll try